gradle/libs.versions.toml

```diff
@@ -1,23 +1,23 @@
 [versions]
 
-kotlin = "1.8.10"
-gradle = "8.0.0"
+kotlin = "1.9.10"
+gradle = "8.2.0"
 gradle-nexus = "1.1.0"
-androidx-core = "1.9.0"
-androidx-test-rules = "1.4.0"
-camerax = "1.2.3"
-compose-bom = "2023.04.01"
-compose-compiler = "1.4.4"
-appcompat = "1.7.1"
-material = "1.6.1"
-lifecycle = "2.6.1"
-navigation = "2.5.2"
-accompanist = "0.25.1"
-coil = "2.2.2"
+androidx-core = "1.12.0"
+androidx-test-rules = "1.5.0"
+camerax = "1.3.1"
```

Comment:
There're some breaking changes on video feature in this version that needs to resolve 😢 

I will try to check this later, but please feel free to resolve or rollback to 1.2.3 again 😄

Reply:
I see you opened up a PR to address this. I rolled back the version here, but lmk if you want me to keep it bumped for easier merging into your branch

Reply:
Hi! Thanks for clarifying things for me!

I've updated, and soon I'll create a new release with our changes. Thanks for your contributions and patience on this.